examples/warehouse/domain/README

```diff
@@ -41,3 +45,10 @@ dns.yaml:
       ns:
         - IP address of NS server authoritative for this domain
         - ...
+
+services.yaml:
+  domain:
+    services:
```

Comment:
The `net.service.` name space would make sense. Also palletjack generally uses singular forms in name space tags.